_rules/video-captions-f51b46.md

```diff
@@ -20,6 +20,8 @@ accessibility_requirements:
     failed: not satisfied
     passed: further testing needed
     inapplicable: further testing needed
+  wcag20:1.2.1: # Audio-only and Video-only (Prerecorded) (A)
+    secondary: true
```

Comment:
```suggestion
    secondary: Because this rule does not consider that the video may be a media alternative for text, it is stricter than the requirement. Failing this rule often results in this success criterion being not satisfied.
```

Or something like that. 

Using https://github.com/act-rules/act-rules.github.io/pull/2060/files as a guide.

Reply:
Changed this text to be the same was what we ended up deciding in the 2060 PR.